File: src/main/java/calebzhou/rdi/microservice/model/json/weather/CaiyunWeather.java

package calebzhou.rdi.microservice.model.json.weather;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class CaiyunWeather {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("api_version")
    @Expose
    private String apiVersion;
    @SerializedName("api_status")
    @Expose
    private String apiStatus;
    @SerializedName("lang")
    @Expose
    private String lang;
    @SerializedName("unit")
    @Expose
    private String unit;
    @SerializedName("tzshift")
    @Expose
    private Integer tzshift;
    @SerializedName("timezone")
    @Expose
    private String timezone;
    @SerializedName("server_time")
    @Expose
    private Integer serverTime;
    @SerializedName("location")
    @Expose
    private List<Double> location = null;
    @SerializedName("result")
    @Expose
    private Result result;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getApiVersion() {
        return apiVersion;
    }

    public void setApiVersion(String apiVersion) {
        this.apiVersion = apiVersion;
    }

    public String getApiStatus() {
        return apiStatus;
    }

    public void setApiStatus(String apiStatus) {
        this.apiStatus = apiStatus;
    }

    public String getLang() {
        return lang;
    }

    public void setLang(String lang) {
        this.lang = lang;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public Integer getTzshift() {
        return tzshift;
    }

    public void setTzshift(Integer tzshift) {
        this.tzshift = tzshift;
    }

    public String getTimezone() {
        return timezone;
    }

    public void setTimezone(String timezone) {
        this.timezone = timezone;
    }

    public Integer getServerTime() {
        return serverTime;
    }

    public void setServerTime(Integer serverTime) {
        this.serverTime = serverTime;
    }

    public List<Double> getLocation() {
        return location;
    }

    public void setLocation(List<Double> location) {
        this.location = location;
    }

    public Result getResult() {
        return result;
    }

    public void setResult(Result result) {
        this.result = result;
    }

}

File: src/main/kotlin/calebzhou/rdi/microservice/model/json/weather/CaiyunWeather.kt
package calebzhou.rdi.microservice.model.json.weather

import com.google.gson.annotations.SerializedName


data class CaiyunWeather (

  @SerializedName("status"      ) var status     : String?           = null,
  @SerializedName("api_version" ) var apiVersion : String?           = null,
  @SerializedName("api_status"  ) var apiStatus  : String?           = null,
  @SerializedName("lang"        ) var lang       : String?           = null,
  @SerializedName("unit"        ) var unit       : String?           = null,
  @SerializedName("tzshift"     ) var tzshift    : Double?              = null,
  @SerializedName("timezone"    ) var timezone   : String?           = null,
  @SerializedName("server_time" ) var serverTime : Double?              = null,
  @SerializedName("location"    ) var location   : ArrayList<Double> = arrayListOf(),
  @SerializedName("result"      ) var result     : Result?           = Result()

)